ERC-20/contracts/LeoToken.sol

```diff
@@ -1,17 +1,32 @@
 //SPDX-License-Identifier: Unlicensed
 pragma solidity ^0.8.0;
+import 'hardhat/console.sol';
 
 contract LeoToken {
 
     string public name = 'Leocode Token';
     string public symbol = 'LEO';
     uint8 public decimals = 18;
+    address public owner;
+
+    uint32 public releasePeriod = 7 days;
+    uint8 public tokenRate = 100;
```

Comment:
Skoro nigdzie nie zmieniasz tych zmiennych, warto się pokusić jednak, żeby to były zmienne `const`